README.asciidoc

```diff
@@ -68,7 +68,7 @@ Same way you can create a test for Kubernetes, Docker, Openshift.
 | arquillian-container-configuration | To configure container configuration.                                            |
 | arquillian-create-test             | To create a test inside package.                                                 |
 | arquillian-cube-setup              | To add arquillian cube dependency & updates arquillian configuration.            |
-| arquillian-cube-create-test        | To configure a test with required Test enricher depending on selected type.      |
+| arquillian-cube-add-test           | To configure a test with required Test enricher depending on selected type.      |
```

Comment:
I think the description of what this command does can be more accurate. We don't really configure the test. We should describe what this command really does.